app-web/src/components/Home/ResourcePreview.js

```diff
@@ -38,43 +40,179 @@ export const ResourceContainer = styled.div`
   }
 `;
 
+const PreviewHeader = styled.div`
+  padding: 10px 4px;
+  display: flex;
+  flex-flow: row wrap;
+  border-bottom: 1px solid #ccc;
+  h2 {
+    font-weight: 400;
+    text-transform: capitalize;
+    padding: 0 20px 0 0;
+    margin: 0 2px;
+    margin-top: 5px;
+  }
+`;
+
+const SeeMoreP = styled.p`
+  text-decoration: underline;
+  color: #1a5a96;
+  font-weight: 400;
+  text-transform: capitalize;
+  padding: 0 4px;
+  margin: 0 2px;
+  :hover {
+    cursor: pointer;
+  }
+`;
+
+const ResourcePill = styled(Pill)`
+  :hover {
+    background: white;
+    border-width: 1px;
+    border-style: solid;
+    border-color: #e0e0e0;
+    top: -3px;
+  }
+`;
+
+const toggled = css`
+  background: white;
+  border-width: 1px;
+  border-style: solid;
+  border-color: #e0e0e0;
+`;
+
 // used by react-testing-library dom querying
 export const TEST_IDS = {
   container: 'resource-preview-container',
 };
 
 // this is a wrapper component that encapsulates cards for topics or other sizes
-export const ResourcePreview = ({ title, link, resources }) => (
-  <Container data-testid={TEST_IDS.container}>
-    <Title>
-      <StyledLink to={link.to}>{title}</StyledLink>
-    </Title>
-    <ResourceContainer>
-      {resources.slice(0, 6).map(r => (
-        <CardWrapper key={r.id}>
-          <Card
-            type={r.fields.resourceType}
-            title={r.fields.title}
-            description={r.fields.description}
-            image={r.fields.image}
-            link={r.fields.standAlonePath}
-            event={r}
+export const ResourcePreview = ({ title, link, resources, filters, amountToShow, seeMore }) => {
+  let [showCount, updateCount] = useState(amountToShow);
```

Comment:
really good, the convention for naming your 'setter' function with `useState` is 'setCount', 'setSeeMore'